README.md

```diff
@@ -1,17 +1,141 @@
-# Project
+# GitHub Action for deploying to Azure Spring Cloud
+GitHub Actions support an automated software development lifecycle workflow. With GitHub Actions for Azure Spring Cloud you can create workflows in your repository to manage your deployment of Azure Spring Cloud conveniently.
 
-> This repo has been populated by an initial template to help get you started. Please
-> make sure to update the content to build a great experience for community-building.
+[comment]: <> (This repository contains GitHub Action for Azure Spring Cloud to deploy. The action supports deploying *.jar, *.war, and *.zip files, or a folder.)
 
-As the maintainer of this project, please make a few updates:
+[comment]: <> (As the maintainer of this project, please make a few updates:)
 
-- Improving this README.MD file to provide a great experience
-- Updating SUPPORT.MD with content about this project's support experience
-- Understanding the security reporting process in SECURITY.MD
-- Remove this section from the README
+[comment]: <> (- Improving this README.MD file to provide a great experience)
 
-## Contributing
+[comment]: <> (- Updating SUPPORT.MD with content about this project's support experience)
+
+[comment]: <> (- Understanding the security reporting process in SECURITY.MD)
+
+[comment]: <> (- Remove this section from the README)
+## Prerequisites
+### Set up GitHub repository and authenticate
+You need an Azure service principal credential to authorize Azure login action. To get an Azure credential, execute the following commands on your local machine:
+```azurecli
+az login
+az ad sp create-for-rbac --role contributor --scopes /subscriptions/<SUBSCRIPTION_ID> --sdk-auth
+```
+
+To access to a specific resource group, you can reduce the scope:
+
+```azurecli
+az ad sp create-for-rbac --role contributor --scopes /subscriptions/<SUBSCRIPTION_ID>/resourceGroups/<RESOURCE_GROUP> --sdk-auth
+```
+The command should output a JSON object:
+
+```json
+{
+    "clientId": "<GUID>",
+    "clientSecret": "<GUID>",
+    "subscriptionId": "<GUID>",
+    "tenantId": "<GUID>",
+    ...
+}
+```
+
+### Dependencies on other GitHub Actions
+* [Checkout](https://github.com/actions/checkout) Checkout your Git repository content into GitHub Actions agent.
+* Authenticate using the [Azure Login Action](https://github.com/Azure/login) with the Azure service principal credential prepared as mentioned above. Examples are given later in this article.
+
+## End-to-End Sample Workflows
+### Deleting a staging deployment
```

Comment:
Shall we re-arrange the order to:
1. deploy
2. set production
3. blue-green
4. delete

Reply:
np

Reply:
per talk offline, the order should be 1, 3, 2, 4